server/migrations/20190514080324-rename-threats-findings-columns.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    await queryInterface.renameColumn('FormHerptiles', 'threatsHerptilesBg', 'findingsBg')
+    await queryInterface.renameColumn('FormHerptiles', 'threatsHerptilesEn', 'findingsEn')
+    await queryInterface.renameColumn('FormMammals', 'threatsMammalsBg', 'findingsBg')
+    await queryInterface.renameColumn('FormMammals', 'threatsMammalsEn', 'findingsEn')
+    await queryInterface.renameColumn('FormInvertebrates', 'threatsInvertebratesBg', 'findingsBg')
+    await queryInterface.renameColumn('FormInvertebrates', 'threatsInvertebratesEn', 'findingsEn')
```

Comment:
@GPopgeorgiev is this correct :point_up:? The existing field `threats` will not be touched, just moved to be populated on each record (mobile) and bottom (web)